### ISSUE TITLE
Open Source deckboard

### ISSUE THREAD

**Issue by K4ds3** [rivafarabi/deckboard#222]
It would be really awesome if the Desktop software & app would be open source, so you can verify the code, build it yourself and help you with fixing bugs 😃

**Comment by TheRolfFR** [comment]
Yes, I would like too! I am a electron developer and the app is great unfortunately the code isn't. I could make a redesign of the app.
I wanted to make a node js server of the app in order to launch it from your phone without an app needed, just start your browser to the page and you have your deck!

**Comment by evilz** [comment]
@TheRolfFR Exactly what I want to do 👍  Compatible with old iPad Gen1

**Comment by terenc3** [comment]
I totally agree that everyone would benefit from open sourcing this app. This would encourage more extension developers to contribute. Currently the missing documentation and inability to check internals is limiting extension potential.

Furthermore it seems @rivafarabi has a lot in backlog and also could use some help to streamline the reported issues in GitHub. Many issues are duplicates or in a bad state, which can be closed.

Please @rivafarabi respond to this request of open sourcing Deckboard and enabling the community to support you, because it is crucial for the feature of Deckboard

**Comment by terenc3** [comment]
@rivafarabi Friendly reminder to answer our questions
